wallet/src/de/schildbach/wallet/ui/BuyAndSellIntegrationsActivity.kt

```diff
@@ -131,11 +129,12 @@ class BuyAndSellIntegrationsActivity : LockScreenActivity(), FancyAlertDialog.Fa
         if (!LiquidConstants.hasValidCredentials() || !UpholdConstants.hasValidCredentials()) {
             binding.keysMissingError.isVisible = true
         }
+        binding.dashServicesList.itemAnimator = null
```

Comment:
Items on the "Buy & Sell" screen have a weird blinking animation when updated with new info, which seems unnecessary.

Reply:
the loading dialog is leftover from the third-party developers who d\id the Liquid Integration.